11ty/definitions/colorblind.md

```diff
@@ -0,0 +1,48 @@
+---
+title: colorblind (race)
+slug: colorblind-racism
+flag:
+  text: 'Medical appropriation'
+  level: avoid
+defined: true
+excerpt: Colorblind, someone with decreased ability to see color or differences in color; colloquially, belief that racism is no longer a problem and that we all have equal opportunities.
+speech: adjective
+alt_words:
+  - ignorant
+  - insensitive
+  - post-racial
+  - race neutral
```

Comment:
If someone wants to talk about policy "race neutral" would be the appropriate term (even if it's inaccurate). "Post-racial" is a fantasy-land term which may not have been used ironically. The others apply to a person. Do you see a benefit in including any of these or just remove the section?

Reply:
I think we should remove the section because it begs more question than it provides answers imho. Generally I would advocate for not using this term in at all, unless referring to the medically appropriate term. How do you feel?